src/plugins/intel_gpu/src/kernel_selector/kernels/permute/permute_kernel_bfzyx_to_bfyxz.cpp

```diff
@@ -0,0 +1,184 @@
+﻿// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "permute_kernel_bfzyx_to_bfyxz.h"
+#include "kernel_selector_utils.h"
+#include "common_tools.h"
+#include <string>
+#include <functional>
+#include <cmath>
+
+// Tile size : 4x4 or 8x8
+#define MIN_TILE_SIZE 4
+#define DEFAULT_TILE_SIZE 8
+
+namespace kernel_selector {
+
+ParamsKey PermuteKernel_bfzyx_to_bfyxz::GetSupportedKey() const {
+    ParamsKey k;
+    k.EnableInputDataType(Datatype::F16);
+    k.EnableInputDataType(Datatype::F32);
+    k.EnableInputDataType(Datatype::INT8);
+    k.EnableInputDataType(Datatype::UINT8);
+    k.EnableInputDataType(Datatype::INT32);
+    k.EnableInputDataType(Datatype::INT64);
+    k.EnableOutputDataType(Datatype::F16);
+    k.EnableOutputDataType(Datatype::F32);
+    k.EnableOutputDataType(Datatype::INT8);
+    k.EnableOutputDataType(Datatype::UINT8);
+    k.EnableOutputDataType(Datatype::INT32);
+    k.EnableOutputDataType(Datatype::INT64);
+    k.EnableDifferentTypes();
+    k.EnableInputLayout(DataLayout::bfzyx);
+    k.EnableOutputLayout(DataLayout::bfzyx);
+    k.EnableTensorOffset();
+    k.EnableTensorPitches();
+    k.EnableBatching();
+    return k;
+}
+
+static inline bool isMultipleDefaultTileSize(const size_t size) {
```

Comment:
nit: `is` -> `Is` The function name starts with upper case in other functions.

Reply:
Updated function name